webrender/src/platform/windows/font.rs

```diff
@@ -40,22 +39,49 @@ fn dwrite_texture_type(render_mode: FontRenderMode) ->
     }
 }
 
-fn dwrite_measure_mode(render_mode: FontRenderMode) ->
+fn dwrite_measure_mode(render_mode: FontRenderMode, options: Option<GlyphOptions>) ->
                        dwrote::DWRITE_MEASURING_MODE {
+    if let Some(option) = options {
```

Comment:
there is quite a few places where you check the options to be `Some()` and the contents to have `force_gdi_rendering == true`. FYI, you could combine the pattern match like this to simplify the code:
```rust
if let Some(GlyphOptions{ force_gdi_rendering: true, .. }) = option {...}
```